relay-general/src/pii/config.rs

```diff
@@ -2,16 +2,41 @@ use std::collections::{BTreeMap, BTreeSet};
 use std::fmt;
 use std::ops::Deref;
 
+use failure::Fail;
 use once_cell::sync::OnceCell;
 use regex::{Regex, RegexBuilder};
 use serde::{de::Error, Deserialize, Deserializer, Serialize, Serializer};
 
 use crate::pii::{CompiledPiiConfig, Redaction};
 use crate::processor::SelectorSpec;
 
+const COMPILED_PATTERN_MAX_SIZE: usize = 262_144;
```

Comment:
this max size was determined by trial and error so that we are able to parse builtin rules at all. I think it was either the creditcard or ip addr regex that required increasing the limit